packages/console/src/assets/docs/fragments/_sign_in_explanation.md

```diff
@@ -0,0 +1,20 @@
+Before we dive into the details, here's a quick overview of the end-user experience. The sign-in process can be simplified as follows:
+
+```mermaid
+graph LR
+    A(Your app) -->|1. Invoke sign-in| B(Logto)
+    B -->|2. Finish sign-in| A
+```
+
+1. Your app invokes the sign-in method.
+2. The user is redirected to the Logto sign-in page. For native apps, the system browser is opened.
```

Comment:
System browser sounds tricky, e.g. iOS popup webview is not the same as opening a Safari browser?

Reply:
it's actually the system browser. iOS only allows the system browser to be used.

Reply:
all webviews, safari controllers, even the Chrome app in iOS use the same browser kernel